src/current/v23.2/cutover-replication.md

```diff
@@ -0,0 +1,174 @@
+(1 row)
+~~~
+
+Specify a timestamp:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER VIRTUAL CLUSTER standbyapplication COMPLETE REPLICATION TO SYSTEM TIME '-1h';
+~~~
+
+Refer to [Using different timestamp formats]({% link {{ page.version.version }}/as-of-system-time.md %}#using-different-timestamp-formats) for more information.
+
+Similarly, to cut over to a specific time in the future:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER VIRTUAL CLUSTER standbyapplication COMPLETE REPLICATION TO SYSTEM TIME '+5h';
+~~~
+
+A future cutover will proceed once the replicated data has reached the specified time.
+
+## Step 2. Completing the cutover
+
+1. The completion of the replication is asynchronous; to monitor its progress use:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTER standbyapplication WITH REPLICATION STATUS;
+    ~~~
+    ~~~
+      id |        name         |         data_state          | service_mode | source_tenant_name |                                                source_cluster_uri                                                 | replication_job_id |       replicated_time        |         retained_time         |          cutover_time
+    -----+---------------------+-----------------------------+--------------+--------------------+-------------------------------------------------------------------------------------------------------------------+--------------------+------------------------------+-------------------------------+---------------------------------
+      4  | standbyapplication  | replication pending cutover | none         | application        | postgresql://user:redacted@3ip:26257/?options=-ccluster%3Dsystem&sslmode=verify-full&sslrootcert=redacted         | 903895265809498113 | 2023-09-28 17:41:18.03092+00 | 2023-09-28 16:09:04.327473+00 | 1695922878030920020.0000000000
+    (1 row)
+    ~~~
+
+    Refer to [Physical Cluster Replication Monitoring]({% link {{ page.version.version }}/physical-cluster-replication-monitoring.md %}) for more  detail.
+
+1. Once complete, start the standby's virtual cluster with:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER VIRTUAL CLUSTER standbyapplication START SERVICE SHARED;
+    ~~~
+
+    ~~~
+      id |        name         |     data_state     | service_mode
+    -----+---------------------+--------------------+---------------
+      1  | system              | ready              | shared
+      2  | template            | ready              | none
+      3  | standbyapplication  | ready              | shared
+    (3 rows)
+    ~~~
+
+1. To make the standby's virtual cluster the default for connection strings. Set the following [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}):
+
```

Comment:
It is worth pointing out that PCR is not enabled on the newly-cut-over now-primary cluster. You'll need to start back at the beginning with a new standby cluster to re-enable it. I guess you do discuss it in the section below but it might be worth linking to there from here.

Reply:
I clarified this a little bit at the end of the Complete Cutover step, hopefully that captures what you were thinking.